src/app/helpers/whitelistApis.js

```diff
@@ -277,6 +278,10 @@ const API_LIST = {
       checksNeeded: ['ROLE_CHECK'],
       ROLE_CHECK: [ROLE.PUBLIC]
     },
+    '/learner/org/v2/preferences/read': {
+      checksNeeded: ['ROLE_CHECK'],
+      ROLE_CHECK: [ROLE.COURSE_MENTOR]
```

Comment:
ROLE_CHECK: [ROLE.PUBLIC] This API should be publicly possible
This is for tenant preferenceApi